tests/app/upload/test_rest.py

```diff
@@ -0,0 +1,180 @@
+
+    service_id = sample_template.service.id
+
+    resp_json = admin_request.get('upload.get_uploads_by_service', service_id=service_id)
+    data = resp_json['data']
+    assert len(data) == 5
+    assert data[0] == {'id': str(upload_5.id),
+                       'original_file_name': 'some.csv',
+                       'notification_count': 10,
+                       'created_at': upload_5.created_at.strftime("%Y-%m-%d %H:%M:%S"),
+                       'statistics': [],
+                       'upload_type': 'job'}
+    assert data[1] == {'id': str(upload_4.id),
+                       'original_file_name': 'some.csv',
+                       'notification_count': 1,
+                       'created_at': upload_4.created_at.strftime(
+                           "%Y-%m-%d %H:%M:%S"),
+                       'statistics': [],
+                       'upload_type': 'job'}
+    assert data[2] == {'id': str(upload_3.id),
+                       'original_file_name': "file-name",
+                       'notification_count': 1,
+                       'created_at': upload_3.created_at.strftime("%Y-%m-%d %H:%M:%S"),
+                       'statistics': [{'count': 1, 'status': 'delivered'}],
+                       'upload_type': 'letter'}
+    assert data[3] == {'id': str(upload_2.id),
+                       'original_file_name': "some.csv",
+                       'notification_count': 1,
+                       'created_at': upload_2.created_at.strftime(
+                           "%Y-%m-%d %H:%M:%S"),
+                       'statistics': [],
+                       'upload_type': 'job'}
+    assert data[4] == {'id': str(upload_1.id),
+                       'original_file_name': "file-name",
+                       'notification_count': 1,
+                       'created_at': upload_1.created_at.strftime("%Y-%m-%d %H:%M:%S"),
+                       'statistics': [{'count': 1, 'status': 'delivered'}],
+                       'upload_type': 'letter'}
+
+
+def test_get_uploads_should_return_statistics(admin_request, sample_template):
+    now = datetime.utcnow()
+    earlier = datetime.utcnow() - timedelta(days=1)
+    job_1 = create_job(template=sample_template, job_status='pending')
+    job_2 = create_job(sample_template, processing_started=earlier)
+    for _ in range(0, 3):
+        create_notification(template=sample_template, job=job_2, status='created')
+
+    job_3 = create_job(sample_template, processing_started=now)
+    for _ in range(0, 4):
+        create_notification(template=sample_template, job=job_3, status='sending')
+    letter_template = create_precompiled_template(sample_template.service)
+    letter_1 = create_uploaded_letter(letter_template, sample_template.service, status='delivered',
+                                      created_at=datetime.utcnow() - timedelta(days=3))
+    letter_2 = create_uploaded_letter(letter_template, sample_template.service, status='delivered',
```

Comment:
Optional: I don't think we are getting any benefit by having `letter_2` in this test. We are already testing a delivered letter by testing the statistics for `letter_1`. Also removing it might help make the set up of this test a bit more readable, a few more line breaks between groups of related lines might also help.